python/hopsworks_common/core/variable_api.py

```diff
@@ -94,16 +100,23 @@ def get_flyingduck_enabled(self) -> bool:
         """
         return self.get_variable("enable_flyingduck") == "true"
 
-    def get_loadbalancer_external_domain(self) -> str:
-        """Get domain of external loadbalancer.
+    def get_loadbalancer_external_domain(self, service: str) -> str:
+        """Get domain of external loadbalancer for MySQLd.
```

Comment:
```suggestion
        """Get domain of external loadbalancer for a service.
```

Reply:
Since my understanding is this function will be used for all services with loadbalancers and not just MySQL